src/loader/timescale_bgw.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TIMESCALE_BGW_H
+#define TIMESCALE_BGW_H
+
+#include <postgres.h>
+#include <fmgr.h>
+
+#define TIMESCALE_MAX_WORKERS_GUC_STANDIN 8
+
+/* Interface for both the loader and the versioned extension*/
```

Comment:
Will the loader itself use this? If not change this to say that this is the interface the versioned extension uses to call functions in the loader. Explain why you use the PG_FUNCTION convention and document how the versioned_extension can call in here (load_external_function etc.). Also maybe loader_api.h is a better name? not sure.

Reply:
The loader uses it in one place, but I think perhaps that is not the way to do it. We should expose the c functions, so that they are easier to call, for use in the loader and make pass throughs using the PG_FUNCTION_INFO_V1 that will live here for use in the versioned extension.